cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/CruiseControlApiImpl.java

```diff
@@ -83,8 +85,8 @@ public static HTTPHeader getAuthHttpHeader(boolean apiAuthEnabled, Secret apiSec
     public Future<CruiseControlResponse> getCruiseControlState(String host, int port, boolean verbose, String userTaskId) {
 
         String path = new PathBuilder(CruiseControlEndpoints.STATE)
-                .addParameter(CruiseControlParameters.JSON, "true")
-                .addParameter(CruiseControlParameters.VERBOSE, String.valueOf(verbose))
+                .withParameter(CruiseControlParameters.JSON, "true")
```

Comment:
I don't really have a problem with changing the name, but I just wonder why?

Reply:
Personally but even in general I see a "builder" made by `withXXXX` methods (as we are using in the rebalance options for example). Was there any specific reasons you used `addXXXX`?